packages/airnode-deployer/src/handlers/aws/index.ts

```diff
@@ -15,8 +25,22 @@ export async function startCoordinator() {
   return { statusCode: 200, body: encodeBody(response) };
 }
 
-export async function initializeProvider(event: any) {
-  const stateWithConfig = { ...event.state, config: parsedConfig };
+export async function run(payload: WorkerPayload): Promise<AWSLambda.APIGatewayProxyResult> {
+  switch (payload.functionName) {
+    case 'initializeProvider':
+      return initializeProvider(payload);
+    case 'callApi':
+      return callApi(payload);
+    case 'processTransactions':
+      return processTransactions(payload);
+  }
+  return { statusCode: 400, body: JSON.stringify({ error: 'Unknown function', payload }) };
```

Comment:
I know this is an unreachable statement according to TS, but since it's a payload of a serverless function that can contain basically anything I'd leave it there.

Reply:
Not sure I understand. How is it posible to cintain anything? It's ustriggering the lambda functions

Reply:
Yes, it is, but it's not like calling a typed function, TS won't help us much. If we make a mistake and send an invalid function name nothing will catch that really. That's why I want to keep the check.

Reply:
I see. But you can cast the function name so that TS knows its type. I think having TS tell you to update this function when you add a new lambda its better to just preventing invalid fn name